src/components/TheButton.tsx

```diff
@@ -97,7 +107,6 @@ export const TheButton = ({
 			onClick={onStart}
 			aria-label="Start timer"
 			disabled={status === 'running'}
-			opacity={status === 'running' ? 0 : 1}
```

Comment:
This is a styled-system prop and already works, we shouldn't need to reimplement it manually.